src/sctools/platform.py

```diff
@@ -418,6 +420,52 @@ def merge_count_matrices(cls, args: Iterable[str]=None) -> int:
         count_matrix.save(args.output_stem)
 
         return 0
+    
+    def group_qc_outputs():
+        """Commandline entrypoint for parsing picard metrics files, hisat2 and rsem statistics log files.
+        Parameters
```

Comment:
I think the right numpydoc block for this is: 

```python
"""
Command line entry-point for parsing Picard, HISAT2, and RSEM statistics log files.

        Returns
        ----------
        int:
            return 0 if the program completes successfully.
"""
```

Reply:
changed